docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -109,7 +109,99 @@ var flexLayout = new FlexLayout();
+
+When positioned absolutely an element doesn't take part in the normal layout flow.
+It is instead laid out independent of its siblings. The position is determined based on the position property.
+
+The following figure shows how the second item works with `FlexPositionType` property:
+
+![FlexPositionType](./media/flexPositionType.png)
+
+## FlexAspectRatio property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexAspectRatio` property has the following properties:
+
+ - Accepts any floating point value > 0, the default is undefined.
+ - Defined as the ratio between the width and the height of a node e.g. if a node has an aspect ratio of 2 then its width is twice the size of its height.
+ - Respects the min and max dimensions of an item.
+ - Has higher priority than `FlexGrow` property.
+ - If aspect ratio, width, and height are set then the cross axis dimension is overridden.
+
+## FlexBasis property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
+Setting the flex basis of a child is similar to setting the width of that child if its parent is a container with flex direction: row or setting the height of a child if its parent is a container with flex direction: column.
+The flex basis of an item is the default size of that item, the size of the item before any flex grow and flex shrink calculations are performed.
+
+## FlexShrink property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflow the size of the container on the main axis.
+The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. These two properties also work well together by allowing children to grow and shrink as needed.
+
+The flex shrink accepts any floating point value >= 0, with 1 being the default value. A container will shrink its children weighted by the child’s flex shrink value.
+
+The following figure shows how the second item works with `FlexShrink` property:
+
+![FlexShrink](./media/flexShrink.png)
+
+## FlexGrow property
+
+> [!NOTE]
+> This property is added in Tizen 6.0.
+
+The `FlexGrow` property describes how any space within a container should be distributed among its children along the main axis. After laying out its children, a container will distribute any remaining space according to the flex grow values specified by its children.
+
+The flex grow accepts any floating point value >= 0, with 0 being the default value. A container will distribute any remaining space among its children weighted by the child’s flex grow value.
+
+The following figure shows how the second item works with `FlexGrow` property:
+
+![FlexGrow](./media/flexGrow.png)
 
 ## Related Information
```

Comment:
Please ensure to follow sentence case for all headings.

Reply:
Done.